documentation/book/con-cluster-operator.adoc

```diff
@@ -0,0 +1,21 @@
+[id='cluster-operator-{context}']
+= Overview of Cluster Operator
+
+The Cluster Operator is in charge of deploying a Kafka cluster alongside a Zookeeper ensemble. As part of the Kafka cluster, it can also deploy the Topic Operator which provides operator-style topic management via ConfigMaps. 
+
+ifdef::Kubernetes[]
+The Cluster Operator can also deploy a Kafka Connect cluster which connects to an existing Kafka cluster. 
+endif::Kubernetes[]
+
+On OpenShift, you can create a cluster and connect to an existing Kafka cluster using the Source2Image feature, which provides an easy way of including more connectors.
```

Comment:
I was at least originally using `{OpenShiftName}` to be able to replace the name if needed (such as OpenShift upstream versus OpenShift Container Platform downstream). Not sure whether we are still using it or not.

Reply:
Yes, @dmuntima we should be using the attribute for OpenShift.